src/types/boxed.rs

```diff
@@ -0,0 +1,343 @@
+use std::any::{self, Any};
+use std::ops::Deref;
+
+use neon_runtime::raw;
+use neon_runtime::external;
+
+use crate::context::{Context, FinalizeContext};
+use crate::context::internal::Env;
+use crate::handle::{Managed, Handle};
+use crate::types::internal::ValueInternal;
+use crate::types::Value;
+
+type BoxAny = Box<dyn Any + Send + 'static>;
```

Comment:
I don't think it makes sense to add parenthesis. These are trait bounds, so they all apply equally. `dyn` is the "optional" specifier that was added in Rust 2018.

In my opinion, adding parenthesis would be less idiomatic.

Reply:
So that means it's `dyn (Any + Send + 'static)`? That's fine if it's not idiomatic to parenthesize. I'd just like to understand.

Reply:
Parenthesis can't change the way that these are applied. It's always all 3 on this simple type. I think with parenthesis it would read `(dyn (Any + Send)) + 'static`.

The only time you typically see parenthesis is with closures to distinguish whether a lifetime is associated with the return type or the closure.